rocAL/rocAL/source/meta_data/meta_node_crop_resize.cpp

```diff
@@ -29,7 +29,7 @@ void CropResizeMetaNode::initialize()
     _y2_val.resize(_batch_size);
 }
 
-void CropResizeMetaNode::update_parameters(MetaDataBatch* input_meta_data)
+void CropResizeMetaNode::update_parameters(pMetaDataBatch input_meta_data, pMetaDataBatch output_meta_data)
```

Comment:
mem copy is not removed in this function

Reply:
Done